R/tuneMBO.R

```diff
@@ -23,15 +28,33 @@ tuneMBO = function(learner, task, resampling, measures, par.set, control,
     messagef("Resuming previous MBO run using state in '%s'...", state)
     or = mlrMBO::mboContinue(state)
   } else {
-    or = mlrMBO::mbo(tff, design = control$mbo.design, learner = control$learner, control = mbo.control, show.info = FALSE)
+    or = mlrMBO::mbo(tff, design = control$mbo.design, learner = control$learner,
+      control = mbo.control, show.info = FALSE)
+  }
+
+  if (multicrit) {
+    df = convertListOfRowsToDataFrame(or$pareto.set)
```

Comment:
That looks strange. First you convert a list into a `df` an in the following line you call `as.list` on each row? Can't you just lapply on `or$pareto.set`?